GAE/src/org/waterforpeople/mapping/app/web/rest/PlacemarkRestService.java

```diff
@@ -50,6 +51,7 @@
     @RequestMapping(method = RequestMethod.GET, value = "")
     @ResponseBody
     public Map<String, Object> listPlaceMarks(
+            @RequestParam(value = "surveyId", defaultValue = "-1") Long surveyId, // default to non-existing surveyId
```

Comment:
How about to extarct this "-1" to a constant something like NON_EXISTING_SURVEY_ID and then no need for comment